### DISCUSSION TITLE
86Box mouse improvement?

### DISCUSSION THREAD

**Discussion by Crinale0** [86Box/86Box#5039] [Q&A] [answered]
Hey all

I'm looking to improve the mouse pointer in 86Box as at the moment it feels sort of laggy? It's not smooth like a normal mouse and I'm trying to run old RTS games such as command and conquer and fun games like Theme Hospital. Both of these rely heavily on the mouse but its just a very jerky experience at the moment.

Any way to make it smooth/less jittery/laggy?

Thanks for everything you do, 86Box ROCKS!

**Comment by OBattler** [comment, marked as answer]
Try the lastest build on Jenkins - a change was made in late December that makes the mouse (and, from what I've been told, also the output frame rate) considerably smoother.

**Comment by Crinale0** [comment]
Thank you, great news!
